services/spar/test-integration/Util/Scim.hs

```diff
@@ -72,6 +73,11 @@ registerScimToken teamid midpid = do
 randomScimUser :: MonadRandom m => m (Scim.User.User SparTag)
 randomScimUser = fst <$> randomScimUserWithSubject
 
+randomUserName :: MonadRandom m => m Text
+randomUserName = do
```

Comment:
This isn't really used anywhere is it?

Reply:
Nope not anymore; I now use the existing randomUser and extract the name instead